### PR TITLE
Logs cleanup

### PR DESCRIPTION
**Pull request by nymkappa**
Fixes https://github.com/mempool/mempool/issues/2332

I've added some tags for most important message but that's a work that will need to be continued over time. @softsimon @mononaut please use the new logger tags for your new logs (at least for >= INFO levels). You can find/add loggers tag in `Logger::tags` object, then you use use it like `logger.debug(`This is a LN related log`, logger.tags.ln);`

~@softsimon Do you have a better approach to store/use logger tags? For now I've just added them in [Logger.tags object](https://github.com/mempool/mempool/pull/2339/files#diff-311a550eb2e3c2360d5611f3cad0e0170c755cf71b46094652c96e61f8c6ed92R35) and I use it like [```logger.info(`${logger.tags.ln} This is a test`);```](https://github.com/mempool/mempool/pull/2339/files#diff-afe74c642a8770eaa6f6b83c0e207e308c2288b0b3c37a1b310d86bbf12631b2R22)~ Fixed

Also I moved lot of lightning spam log into `debug` and only kept `info` for getting progress update (every 30 seconds by default, configurable with `config.LIGHTNING.LOGGER_UPDATE_INTERVAL`) and also update on nodes/channels.

Example while running locally:

```
Nov 23 20:07:07 [8709] NOTICE: <lightning> Starting Mempool Server... (c863a74)
Nov 23 20:07:07 [8709] INFO: <lightning> Restoring mempool and blocks data from disk cache
Nov 23 20:07:07 [8709] INFO: <lightning> CLightning client connected
Nov 23 20:07:07 [8709] INFO: <lightning> Database connection established.
Nov 23 20:07:07 [8709] INFO: <lightning> Starting statistics service
Nov 23 20:07:07 [8709] INFO: <lightning> Starting currency rates service
Nov 23 20:07:07 [8709] INFO: <lightning> Currency rates service will be queried over clearnet using https://price.bisq.wiz.biz/getAllMarketPrices
Nov 23 20:07:07 [8709] NOTICE: <lightning> Mempool Server is running on port 8999
Nov 23 20:07:07 [8709] INFO: <lightning> [Mining] Updating latest mining pools from https://raw.githubusercontent.com/mempool/mining-pools/master/pools.json over clearnet
Nov 23 20:07:07 [8709] INFO: <lightning> [Lightning] Starting lightning network sync service
Nov 23 20:07:10 [8709] INFO: <lightning> [Mining] Fetching daily price history from exchanges and saving missing ones into the database
Nov 23 20:07:16 [8709] NOTICE: <lightning> [Mining] Inserted 1 daily historical prices into the db
Nov 23 20:07:16 [8709] INFO: <lightning> [Mining] Fetching hourly price history from exchanges and saving missing ones into the database
Nov 23 20:07:22 [8709] INFO: <lightning> [Mining] Indexing 8200 blocks from #754400 to #744401
Nov 23 20:07:30 [8709] INFO: <lightning> [Lightning] Starting closed channels scan for the first time
Nov 23 20:07:38 [8709] INFO: <lightning> [Lightning] Checking if channel has been closed 7713/111122
Nov 23 20:07:53 [8709] NOTICE: <lightning> The mempool is now in sync!
Nov 23 20:08:08 [8709] INFO: <lightning> [Lightning] Checking if channel has been closed 18556/111122
Nov 23 20:08:39 [8709] INFO: <lightning> [Lightning] Checking if channel has been closed 30007/111122
```

### CODE REVIEW

**Comment by softsimon** [comment]
Logger tags is a good idea. But it would be better to add it as a separate input parameter to the logger method and having it added later, with a default setting. Then we could disable/enable logging or set log levels of specific tags.

```
logger.debug(`This is a test`, logger.tags.ln);
```

File: backend/src/api/lightning/clightning/clightning-client.ts [hunk 1]
```diff
@@ -141,13 +141,13 @@ export default class CLightningClient extends EventEmitter implements AbstractLi
...
-        logger.warn(`[CLightningClient] ${rpcPath}/lightning-rpc is missing, using the bitcoin mainnet subdirectory at ${rpcPath}/bitcoin instead.`)
-        logger.warn(`[CLightningClient] specifying the main lightning data directory is deprecated, please specify the network directory explicitly.\n`)
+        logger.warn(`${rpcPath}/lightning-rpc is missing, using the bitcoin mainnet subdirectory at ${rpcPath}/bitcoin instead.`, logger.tags.ln)
+        logger.warn(`specifying the main lightning data directory is deprecated, please specify the network directory explicitly.\n`, logger.tags.ln)
         rpcPath = path.join(rpcPath, 'bitcoin', 'lightning-rpc')
       }
     }
 
-    logger.debug(`[CLightningClient] Connecting to ${rpcPath}`);
+    logger.debug(`Connecting to ${rpcPath}`), logger.tags.ln;
```
Comment (junderw) [inline backend/src/api/lightning/clightning/clightning-client.ts:150]: The same mistake appears multiple times in this file.

```suggestion
    logger.debug(`Connecting to ${rpcPath}`, logger.tags.ln);
```

File: backend/src/tasks/lightning/sync-tasks/funding-tx-fetcher.ts [hunk 2]
```diff
@@ -44,9 +44,9 @@ class FundingTxFetcher {
...
 
       let elapsedSeconds = Math.round((new Date().getTime() / 1000) - loggerTimer);
-      if (elapsedSeconds > 10) {
+      if (elapsedSeconds > config.LIGHTNING.LOGGER_UPDATE_INTERVAL) {
         elapsedSeconds = Math.round((new Date().getTime() / 1000) - globalTimer);
-        logger.info(`Indexing channels funding tx ${channelProcessed + 1} of ${channelIds.length} ` +
+        logger.info(`Indexing channels funding tx ${channelProcessed + 1} of ${channelIds.length} `, logger.tags.ln +
           `(${Math.floor(channelProcessed / channelIds.length * 10000) / 100}%) | ` +
           `elapsed: ${elapsedSeconds} seconds`
         );
```
Comment (junderw) [inline backend/src/tasks/lightning/sync-tasks/funding-tx-fetcher.ts:52]: Log strings with multi-lines are also broken it seems.
```suggestion
        logger.info(`Indexing channels funding tx ${channelProcessed + 1} of ${channelIds.length} ` +
          `(${Math.floor(channelProcessed / channelIds.length * 10000) / 100}%) | ` +
          `elapsed: ${elapsedSeconds} seconds`, logger.tags.ln
        );
```

Comment (nymkappa) [reply backend/src/tasks/lightning/sync-tasks/funding-tx-fetcher.ts:52]: Damn it javascript

File: backend/src/tasks/lightning/network-sync.service.ts [hunk 3]
```diff
@@ -293,18 +296,186 @@ class NetworkSyncService {
...
+          // 'Lightning Anchor';
+          return 7;
+        } else {
+          // top element is '', it has been swept after 16 blocks
+          // 'Swept Lightning Anchor';
+          return 8;
+        }
+      }
+      return 1;
+  }
```
Comment (wiz) [inline backend/src/tasks/lightning/network-sync.service.ts:478]: possible mis-merge?

**Review by softsimon — APPROVED** [review]
code + tested ACK @ efb4827

**Pull request merged**